src/lib/registration/connect_helpers.rb

```diff
@@ -71,9 +71,14 @@ def self.catch_registration_errors(message_prefix: "", show_update_hint: false,
         log.error "Network error: #{e.class}: #{e.message}"
         handle_network_error(message_prefix, e)
         false
-      rescue Timeout::Error
+      rescue Timeout::Error => e
         # Error popup
-        Yast::Report.Error(_("Connection time out."))
+        log.error "Timeout error: #{e.message}"
+        Yast::Report.Error(
+          error_with_details(message_prefix + _("Connection time out.\n"),
```

Comment:
To not break the existing translations I'd prefer keeping the old text, e.g. use `_("Connection time out.") + "\n"` here.